tests/console/sshd.pm

```diff
@@ -47,17 +47,14 @@ sub run {
     # new user to test sshd
     my $ssh_testman        = "sshboy";
     my $ssh_testman_passwd = get_var('PUBLIC_CLOUD') ? random_string(8) : 'let3me2in1';
-    assert_script_run('echo -e "Match User ' . $ssh_testman . '\n\tPasswordAuthentication yes" >> /etc/ssh/sshd_config') if (get_var('PUBLIC_CLOUD'));
 
-    zypper_call('in expect');
+    # Allow password authentication for $ssh_testman
+    assert_script_run(qq(echo -e "Match User $ssh_testman\\n\\tPasswordAuthentication yes" >> /etc/ssh/sshd_config)) if (get_var('PUBLIC_CLOUD'));
 
     # 'nc' is not installed by default on JeOS
-    if (script_run("which nc")) {
-        zypper_call("in netcat-openbsd");
-    }
-    if (script_run("which killall")) {
-        zypper_call("in psmisc");
-    }
+    zypper_call("in netcat-openbsd") if (script_run("which nc"));
+    zypper_call('in expect')         if (script_run("which expect"));
+    zypper_call("in psmisc")         if (script_run("which killall"));
```

Comment:
Why not just `zypper_call('in netcat-openbsd expect psmisc')`
One execution instead of six.

Reply:
You mean like this?
```perl
zypper_call("in netcat-openbsd expect psmisc") if (script_run("which nc expect killall"));
```

Reply:
no, I mean just install it, you don't need to check if it's present

Reply:
But you can keep it as it is, I just think there is no reason to complicate simple things.